tests/cbmc/proofs/s2n_stuffer_alloc/s2n_stuffer_alloc_harness.yaml

```diff
@@ -0,0 +1,10 @@
+batchpkg: cbmc-batch.tar.gz
```

Comment:
This is the wrong .yaml file.  You need the cbmc_batch.yaml

Reply:
Done.